mystery_word.py

```diff
@@ -0,0 +1,129 @@
+import random
+
+
+def new_word():
+    """Imports dictionary, breaks into list, cleans whitespace and lowercases
+    everything. Then returns one random choice word from the list"""
+    with open("/usr/share/dict/words") as word_list:
+        word_list = word_list.readlines()  # breaks into a list of strings
+        word_list = [word.strip().lower() for word in word_list]  # cleans list
+        new_word = random.choice(word_list)
+        return new_word  # as a string
+
+
+def difficulty(level="n"):
```

Comment:
In here, I'd use a list comprehension to get a list of words that meet the difficulty requirements. This would require the changes I suggested in `new_word`, though.